### PR TITLE
[homework #6] - js basics 1 by Edelmiro Antón

### PR DESCRIPTION
**Pull request by EdelmiroAnton**
Hey! @ilya-ischenko

Here are the katas that Mike told us to push so you can take a look. 
Meanwhile I'll continue with the rest and the next hw.

Thanks!

### CODE REVIEW

**Review by ilya-ischenko — COMMENTED** [review]
Looks good as for me!
You can avoid writing obvious comments, 
> like
 if (s.length % 2 === 0) {
    //Check if is the length of the string is even

File: homeworks/edelmiro-anton/06-JS-basics/basics_1.js [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+  let char = strLower[i];
+  let slice = strLower.slice(i + 1);
+
+  if (!char.includes(' ')) {
+    let newStr = substring + char.toUpperCase() + slice;
+    arr.push(newStr);
+  }
+}
+
+console.log(arr);
```
[the remark this answers is not in the document]
Comment (EdelmiroAnton) [reply homeworks/edelmiro-anton/06-JS-basics/basics_1.js]: Fixed

File: homeworks/edelmiro-anton/06-JS-basics/basics_1.js [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+  return counter;
+}
+
+// console.log(countWords(string));
+
+// Task 9 - https://www.codewars.com/kata/find-the-odd-int/
+
+function findOdd(A) {
+  A.sort();
+  let elements = [];
```
[the remark this answers is not in the document]
Comment (EdelmiroAnton) [reply homeworks/edelmiro-anton/06-JS-basics/basics_1.js]: Done at katas 6, 9 & 10

File: homeworks/edelmiro-anton/06-JS-basics/basics_1.js [hunk 3]
```diff
@@ -0,0 +1,281 @@
...
+    }
+  });
+
+  let numbers = map.filter((el) => {
+    // Removes undefined from array
+    return el != undefined;
+  });
+
+  return parseInt(numbers.join(''));
+};
```
[the remark this answers is not in the document]
Comment (EdelmiroAnton) [reply homeworks/edelmiro-anton/06-JS-basics/basics_1.js]: You're right, It's done now

**Review by ilya-ischenko — APPROVED** [review]
Well done!
Approved.

**Pull request merged**